src/main/java/com/example/Goods.java

```diff
@@ -0,0 +1,12 @@
+package com.example;
+
+/**
+ * A goods item is what the customer comes to the shop for.
+ */
+public class Goods {
+
+    /**
+     * The selected item is put into the shopping basket or shopping trolley.
+     */
+    String item;
```

Comment:
I don't get it. If it is plural ("good") then there should be several items. Here we have one.

Also, why does this field have package-private visibility? It must be `private`.

Reply:
Means one item of goods as described in the Javadoc before `Goods`.